models/income.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (sequelize, DataTypes) => {
+    var Income = sequelize.define("Income", {
+    income_source: {type: DataTypes.STRING},
+    plan: {type: DataTypes.INTEGER},
+    spent: {type:DataTypes.INTEGER},
+    created: {type:DataTypes.STRING},
+    modified: {type:DataTypes.STRING},
+    dateid: {type: DataTypes.DATE}
```

Comment:
I think we want the dateid column to be auto-generated when we make the association with the date model. When that association is made it should generate a foreign key.